### DISCUSSION TITLE
Add ability to use external database

### DISCUSSION THREAD

**Discussion by ajtatum** [netalertx/NetAlertX#510] [Ideas]
Sometimes I feel as though the app lags and the database file size is 116,48 MB, so perhaps being able to offload the database away from sqlite and onto either MariaDB or Postgres would offer some benefits?

**Comment by jokob-sk** [comment]
Thanks for the idea!

I'm not looking into supporting more DB deployments as I'm already stretched thin with the current requests and support queries.

Instead I'd look into why the DB size is this big and implement maintenance tasks to keep the DB size in check.

A 100mb size sounds like either a maintenance task is not executed, too much data is retained or there is a bug.

Please open a new issue if you want to help investigate further.

Thanks,
J
